### PR TITLE
doc: Usage and setting spending limits

### PR DESCRIPTION
**Pull request by pilvikala**
Adding docs on how pricing works and how to monitor the usage.

### CODE REVIEW

File: website/pages/docs/billing-and-usage/overview.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Billing and Usage | Overview
+description: Learn about CloudQuery billing model
```
Comment (hermanschaaf) [inline website/pages/docs/billing-and-usage/overview.md:3]:
```suggestion
description: Learn about CloudQuery's billing model
```

File: website/pages/docs/billing-and-usage/spending-limits.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Billing and Usage | Spending Limits
+description: CloudQuery is an open source ELT framework that can be used as an alternative to Airbyte. CloudQuery is built for performance, and is easy to deploy and maintain.
```
Comment (hermanschaaf) [inline website/pages/docs/billing-and-usage/spending-limits.md:3]: We may want to update this description to better reflect the contents of this page

File: website/pages/docs/billing-and-usage/spending-limits.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+# Spending Limits
+
+You can set your maximum spend per month in your billing page by clicking the pencil icon.
+
+![Setting spending limit](/images/docs/billing-and-usage/spending-limit.png)
+
+ When running a sync, CloudQuery CLI will continously check your current spend and once you reach the limit, the sync will end with an error.
```
Comment (hermanschaaf) [inline website/pages/docs/billing-and-usage/spending-limits.md:12]:
```suggestion
 When running a sync, CloudQuery CLI will periodically check your current spend and if you reach the limit, the sync will end with an error.
```

**Review by hermanschaaf — CHANGES_REQUESTED** [review]

File: website/pages/docs/billing-and-usage/spending-limits.md (hunk 2 above)
Comment (pilvikala) [reply website/pages/docs/billing-and-usage/spending-limits.md]: updated

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**